mycroft/skills/roku_control/__init__.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2016 Mycroft AI, Inc.
+#
+# This file is part of Mycroft Core.
+#
+# Mycroft Core is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Mycroft Core is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Mycroft Core.  If not, see <http://www.gnu.org/licenses/>.
+
+from roku import Roku
+from adapt.intent import IntentBuilder
+from os.path import dirname
+from time import sleep
+
+from mycroft.skills.core import MycroftSkill
+from mycroft.util.log import getLogger
+
+__author__ = 'eward'
+
+logger = getLogger(__name__)
+
+
+class RokuSkill(MycroftSkill):
+    def __init__(self):
+        super(RokuSkill, self).__init__(name="RokuSkill")
+        self.ip = str(self.config['ip'])
+        self.roku = None
+        if self.ip:
```

Comment:
It seems this skill cannot work without a IP. In that case, we should throw a ValueError exception and prevent this skill to run. So, let's add a `else` for that.